ab_tool/static/ab_tool/js/experiments_dashboard.js

```diff
@@ -9,7 +9,7 @@ $(document).ready(function(){
     });
 
     function is_valid_url(url) {
-        return /^(http(s)?:\/\/)?(www\.)?[a-z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/.test(url);
+        return /^(http(s)?:\/\/)?(www\.)?[a-zA-Z0-9]+([\-\.]{1}[a-z0-9]+)*\.[a-z]{2,5}(:[0-9]{1,5})?(\/.*)?$/.test(url);
     }
 
     $('.addIntervention').modal('show');
```

Comment:
This was already handled as part of another story to fix url validation - it was merged today.. It was done slightly differently - case insensitivity was added across the whole URL . It may need to be merged into this PR?